docs/general/configuration_options.md

```diff
@@ -5,6 +5,7 @@ The following configuration options can be set on `ActiveModel::Serializer.confi
 ## General
 
 - `adapter`: The [adapter](adapters.md) to use. Possible values: `:attributes, :json, :json_api`. Default: `:attributes`.
+- `automatic_lookup`: Whether serializer should be automatically looked up or manually provided. Default: `true`
```

Comment:
@bf4 (hey!)

#1294 addresses [1 of the 2 problem cases](https://github.com/rails-api/active_model_serializers/pull/1294#issuecomment-152831189). I think the root problem is that automatic lookup is too aggressive of a strategy for larger apps that want to introduce AMS only for new versions of an existing API. It'd be nice – and frankly _safer_ – to opt out globally and opt-in locally in these cases.

Another riff on this PR is to make that an explicit flag on render calls:

```
render json: photos, use_serializer: true
```

But I already personally prefer supplying the specific serializer to the render call. Yes, it's repetitive, but removes the "where the heck is this object turning into _this_ JSON?!?" problem.